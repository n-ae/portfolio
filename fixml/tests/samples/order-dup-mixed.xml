  <Project>
	<ItemGroup>
 <PackageReference Include="Test"/>
	<PackageReference Include="Test"/>
  </ItemGroup>
	<PropertyGroup>
 <TargetFramework>net6.0</TargetFramework>
	</PropertyGroup>
  </Project>