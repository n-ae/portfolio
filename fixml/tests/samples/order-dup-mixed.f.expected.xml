<?xml version="1.0" encoding="utf-8"?>
<Project>
  <ItemGroup>
    <PackageReference Include="Test"/>
  </ItemGroup>
  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
  </PropertyGroup>
</Project>
